File: Kotlin/Pitagoras.kt
fun main() {

    val x = readLine()!!.toInt()
    val y = readLine()!!.toInt()

    println("Hipotenusa: ${pitagoras(x, y)}")
}

fun pitagoras(x: Int, y: Int): Double {
    var hip: Double

    hip = Math.sqrt(((x*x) + (y*y)).toDouble())

    return hip
}

File: C#/Pitagoras.cs
using System;
public class Pitagoras{
    public static void Main()
    {
        double b=0;
        double c=0;

        b = double.Parse(Console.ReadLine());
        c = double.Parse(Console.ReadLine());

        double resultado = System.Math.Sqrt(b*b + c*c);

        Console.WriteLine("Hello Hacktoberfest");
        Console.WriteLine(resultado);
    }
}
